cli/run/batchWarnings.ts

```diff
@@ -237,7 +237,13 @@ const deferredHandlers: {
 	UNUSED_EXTERNAL_IMPORT(warnings) {
 		title('Unused external imports');
 		for (const warning of warnings) {
-			stderr(`${warning.names} imported from external module '${warning.source}' but never used`);
+			stderr(
+				`${warning.names} imported from external module '${warning.source}' but never used${
+					warning.sources
```

Comment:
This check is not needed as looking at the only place where the warning is generated, 
https://github.com/rollup/rollup/blob/e88c5c885790305d348c52faf30db835475323a5/src/ExternalModule.ts#L123
`sources` will always be an array. It will further not be empty as exports (`this.declarations`) of external modules are determined by usage.